example-monorepos/with-tailwind/apps/next/global.css

```diff
@@ -1,2 +1,43 @@
+@tailwind base;
 @tailwind components;
 @tailwind utilities;
+
```

Comment:
Can we move this CSS back into `_document` directly? We should discourage `.css` files.

Reply:
Yeah I'm not sure to understand why it's not working as it used to... Using the `global.css` file for this specific use-case seems OK in my opinion. Adding the comment!